src/types/common.d.ts

```diff
@@ -43,3 +53,36 @@ export interface RenderContext {
    */
   measureText(text: string): { width: number };
 }
+
+export interface AccidentalCode {
```

Comment:
I would prefer to keep `Accidental*` and `Line` in `accidental.ts` please.

Reply:
You can also remove the `Accidental` prefix for them in that file (`accidental.ts`.)

Reply:
@0xfe I would not remove the prefix we didn´t do that with the others Element... and I would specially not do it if they are exported.

Reply:
@0xfe related to the above, Accidental will help to associate the dependecies

![image](https://user-images.githubusercontent.com/22865285/114617312-44582c00-9ca8-11eb-88c5-d1f133d97cbe.png)

Reply:
Okay, agreed in this case. Thanks.